research/src/pages/component-spec-template.mdx

```diff
@@ -0,0 +1,213 @@
+> This document represents a component specification template. Its purpose
+> is to provide a point-in-time guide as to how a component should be
+> authored, in light of common anatomies, behaviors, and accessibility.
+> The primary audience is a developer authoring a given component.
+
+# {ComponentName} Component Specification
+
+## Overview
+
+*The name of the component, along with a high-level description.*
+
+### Background
+
+*Relevant historical or background information, related existing issues, etc.*
+
+### Use Cases
+
+*Primary use cases for this component.*
+
+### Non-goals
+
+*A list of use cases, features, or functionality which are **not** goals for the component.*
+  
+### Features
```

Comment:
I see overview as a quick summary, along with background if there is any. Use cases provide the specific scenarios (I could rename that User Stories). Features, however, describes how all of that maps to specific capabilities of the component. A given user case could be accomplished through different conceptualization of features, for example.

Reply:
I guess I'll need to some specific examples because it feels like they maybe should be combined rather than disjointed. Let's not block on this however.